tier4/driving_log_replayer
# feat: shell completion

## Types of PR


## Description

- change cli command
- add document to setup shell completion

## How to review this PR

- Install cli and shell completion scripts.
- Confirm that command completion works with `Tab` key

```shell
pipx uninstall driving-log-replayer-cli
git clone https://github.com/tier4/driving_log_replayer.git
cd driving_log_replayer
git checkout feat/shell-completion
# install cli
pipx install .
# install shell completion

# for bash user
_DLR_COMPLETE=bash_source dlr > $HOME/.dlr-complete.bash
_DLR_COMPLETE=bash_source dlr > $HOME/.dlr-analyzer-complete.bash

echo "source $HOME/.dlr-complete.bash" >> ~/.bashrc
echo "source $HOME/.dlr-analyzer-complete.bash" >> ~/.bashrc

# for fish user
_DLR_COMPLETE=fish_source dlr > $HOME/.config/fish/completions/dlr.fish
_DLR_ANALYZER_COMPLETE=fish_source dlr-analyzer > $HOME/.config/fish/completions/dlr-analyzer.fish

# then command below.
dlr [Space] [Tab]
```

## Others

Completion works like this video.
![completion](https://github.com/tier4/driving_log_replayer/assets/16977736/466c383a-e5f9-4c5d-b483-2c7f3e415ac2)
